app/src/features/projects/components/ProjectFundingItemForm.test.tsx

```diff
@@ -42,16 +44,36 @@ const investment_action_category: IInvestmentActionCategoryOption[] = [
 
 describe('ProjectFundingItemForm', () => {
   it('renders correctly with default empty values', () => {
+    const existingFormValues: IProjectFundingFormArrayItem = {
+      id: 1,
+      agency_id: 1,
+      investment_action_category: 1,
+      investment_action_category_name: 'Something',
+      agency_project_id: '555',
+      funding_amount: 666,
+      start_date: '2021-03-14',
+      end_date: '2021-04-14',
+      revision_count: 2
+    };
+
     const { baseElement } = render(
       <div id="root">
-        <ProjectFundingItemForm
+        <EditDialog
```

Comment:
I dont think this test is testing what you want. You are rendering just the editdialog even though it is the projectfundingitemform component, so this test does not seem correct

Reply:
we should be rendering the component directly within the formik wrapper in these tests, not rendering the edit dialog directly